cmd/anonymizer/app/query/query.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package query
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/proto-gen/api_v2"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+)
+
+// Query represents a jaeger-query's query for trace-id
+type Query struct {
+	api_v2.QueryServiceClient
```

Comment:
nit: it's a good practice to avoid embedding structs like this unless you really want the Query struct to expose methods of QueryServiceClient, which you probably don't. In other words, this grpc client is an implementation detail that does not need to be exposed to users of the Query struct.
```suggestion
	client api_v2.QueryServiceClient
```